dev.sh

```diff
@@ -0,0 +1,2 @@
+mdbook-admonish install --css-dir src/theme
+mdbook serve --port 3000 -n 0.0.0.0
```

Comment:
Missing new line at the end of file
```suggestion
mdbook serve --port 3000 -n 0.0.0.0

```